test/e2e/performanceprofile/functests/7_performance_kubelet_node/cgroups.go

```diff
@@ -76,6 +171,326 @@ var _ = Describe("[performance] Cgroups and affinity", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(ovsLoadBalancing).To(Equal("0"))
 		})
+		It("[test_id:64098] ovs-vswitchd and ovsdb-server are under ovs.slice cgroup", func() {
+			processCgroups := "cpuset:/ovs.slice"
```

Comment:
This is an implementation detail, we do not actually care about this as long as the cpus can escape the reserved cpuset.

So I suggest we update this to check the cpuset cpus of the processes span all available cpus. Also would it make sense to write this in a way that is compatible with the upcoming move to cgroups v2?

Reply:
@MarSik Addressed this in the latest commit